.ci/static-checks.sh

```diff
@@ -299,6 +299,27 @@ static_check_go_arch_specific()
 	eval "$linter" "${linter_args}" "$dirs"
 }
 
+static_check_rust_arch_specific()
+{
+	local -r libs=("rustjail" "oci")
+	local -r bins=(".")
+	for lib in ${libs[@]}; do
+		rustfmt --check ${lib}/src/lib.rs > /dev/null 2>&1
```

Comment:
This depend on the repo this is run  where is run and will work only in agent  (as we expect but should not be this then in the agent repo

Reply:
I don't understand it. directory problem?  

The functions for go also assume that script 's work directory is the repo directory.

Reply:
I think @jcvenegas is saying (and I agree) that you shouldn't be assuming `$dir/src/lib.rs`. Instead, why not do something like:

```sh
find . -type f -name "*.rs" | egrep -v "grpc-rs/|target/" | xargs rustfmt ...
```

Admittedly, that still hard-codes a small amount of rust agent knowledge (the `grpc-rs`), but that problem could be overcome easily by moving `grpc-rs/` into `vendor/grpc-rs/` and updating the new agents `Cargo.toml` to specify:

```toml
grpcio = { path = "vendor/grpc-rs" }
```

Reply:
grpc-rs has dozens of submodules, it might be easier to use it as submodule instead of a vendor directory

Reply:
lib crates have src/lib.rs, bin crates have src/main.rs. I don't understand what the problem is

Reply:
> I think @jcvenegas is saying (and I agree) that you shouldn't be assuming `$dir/src/lib.rs`. Instead, why not do something like:
> 
> ```shell
> find . -type f -name "*.rs" | egrep -v "grpc-rs/|target/" | xargs rustfmt ...
> ```
> 
> Admittedly, that still hard-codes a small amount of rust agent knowledge (the `grpc-rs`), but that problem could be overcome easily by moving `grpc-rs/` into `vendor/grpc-rs/` and updating the new agents `Cargo.toml` to specify:
> 
> ```toml
> grpcio = { path = "vendor/grpc-rs" }
> ```

I see. clone submodule into vendor/grpc-rs.

Reply:
fixed